runtime/tests/runtime_integration_tests/test_update_reset.rs

```diff
@@ -0,0 +1,212 @@
+// Licensed under the Apache-2.0 license
+
+use std::mem::size_of;
+
+use caliptra_builder::{
+    firmware::{self, APP_WITH_UART, FMC_WITH_UART},
+    ImageOptions,
+};
+use caliptra_common::mailbox_api::{
+    CommandId, FwInfoResp, MailboxReq, MailboxReqHeader, TagTciReq,
+};
+use caliptra_error::CaliptraError;
+use caliptra_hw_model::HwModel;
+use caliptra_runtime::RtBootStatus;
+use dpe::{DpeInstance, U8Bool, MAX_HANDLES};
+use zerocopy::{AsBytes, FromBytes};
+
+use crate::common::run_rt_test;
+
+#[test]
+fn test_rt_journey_pcr_updated_in_dpe() {
+    let mut model = run_rt_test(None, None, None);
+
+    model.step_until(|m| {
+        m.soc_ifc().cptra_boot_status().read() == u32::from(RtBootStatus::RtReadyForCommands)
+    });
+
+    // trigger update reset
+    let updated_fw_image = caliptra_builder::build_and_sign_image(
+        &FMC_WITH_UART,
+        &firmware::runtime_tests::MBOX,
+        ImageOptions::default(),
+    )
+    .unwrap()
+    .to_bytes()
+    .unwrap();
+    model
+        .mailbox_execute(u32::from(CommandId::FIRMWARE_LOAD), &updated_fw_image)
+        .unwrap();
+
+    let rt_journey_pcr_resp = model.mailbox_execute(0x1000_0000, &[]).unwrap().unwrap();
```

Comment:
nit: for all the mbox_responder command IDs, could you define consts or an enum somewhere in the test code? I think it will make it easier to well what the tests are doing.

Reply:
I tried to do this earlier, but its a bit hard since mbox_responder lives in a separate crate than the tests, so we would have to define these codes in real RT firmware which isn't preferable. The reason we can do this for FW_LOAD is that it's already a mbox command in RT. 

I agree though and I'll try to refactor the test code in a future PR so that this is possible.